periodic-capi-provider-ibmcoud-e2e-boskos-new is failing

/kind bug
/area provider/ibmcloud

**What steps did you take and what happened:**
periodic-capi-provider-ibmcoud-e2e-boskos-new is failing with following error:
```shell
Install complete.
panic: detect: not detected

goroutine 1 [running]:
github.ibm.com/SoftLayer/softlayer-cli/plugin/i18n.DetectLocal()
	/home/travis/gopath/pkg/mod/github.ibm.com/!soft!layer/softlayer-cli@v1.0.8/plugin/i18n/i18n.go:72 +0x105
github.ibm.com/SoftLayer/softlayer-cli/plugin/i18n.supportedLocale({0x0?, 0x0?})
	/home/travis/gopath/pkg/mod/github.ibm.com/!soft!layer/softlayer-cli@v1.0.8/plugin/i18n/i18n.go:90 +0x54
github.ibm.com/SoftLayer/softlayer-cli/plugin/i18n.Init({0x7ff23aac1a18?, 0xc0001d7aa0?})
	/home/travis/gopath/pkg/mod/github.ibm.com/!soft!layer/softlayer-cli@v1.0.8/plugin/i18n/i18n.go:45 +0x2f
github.ibm.com/SoftLayer/softlayer-cli/plugin/i18n.init()
	/home/travis/gopath/pkg/mod/github.ibm.com/!soft!layer/softlayer-cli@v1.0.8/plugin/i18n/i18n.go:41 +0xd2
+ EXIT_VALUE=2
```
Job logs - https://prow.ppc64le-cloud.org/view/gs/ppc64le-kubernetes/logs/periodic-capi-provider-ibmcoud-e2e-boskos-new/1573515157382893568

**What did you expect to happen:**


**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]


**Environment:**

- Cluster-api version:
- Minikube/KIND version:
- Kubernetes version: (use `kubectl version`):
- OS (e.g. from `/etc/os-release`):

--- a/scripts/ci-e2e.sh
+++ b/scripts/ci-e2e.sh
@@ -18,6 +18,10 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+# With the recent ibmcloud have seen panic and need this environment set to avoid this panic, for more information
+# refer the Notes section in https://github.com/IBM-Cloud/ibm-cloud-cli-release/releases/tag/v2.11.0.
+export LANG=en_US.UTF-8
+
 REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 cd "${REPO_ROOT}" || exit 1
 GOPATH_BIN="$(go env GOPATH)/bin/"

